docs/reference/troubleshooting/data/increase-shard-limit.asciidoc

```diff
@@ -0,0 +1,18 @@
+[[increase-shard-limit]]
+== The configured maximum number of shards that can be allocated to a single node has been reached
```

Comment:
This title is a bit messy. Any better suggestions?

Reply:
Perhaps "Maximum number of shards on a single node exceeded" ?

Reply:
Would that be clear that it's for a particular index? 

Maybe "Maximum number of shards for an index on a single node exceeded" ?